pkgs/by-name/nv/nvidia-modprobe/package.nix

```diff
@@ -0,0 +1,30 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  gnum4,
+}:
+stdenv.mkDerivation rec {
+  pname = "nvidia-modprobe";
+  version = "550.54.14";
```

Comment:
https://github.com/NVIDIA/nvidia-modprobe/releases/tag/565.57.01

Reply:
I just left it as-is for now. Generally, you'd want to package the latest minor/patch for every major version, as the major versions indicate device compatibility. However, that's kind of difficult to do in nixpkgs without special packaging infrastructure like for the drivers.

Reply:
Agreed, hence the merge. The diffs between major versions do not look correlated, so, I'm just glad to have access to this tool.